src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$main-color: #060b35;
+$secondary-color: #616070;
+$blue-color: #00acdc;
```

Comment:
👍  colors are first candidates to go to variables

One point: blue-color is not a perfect name, as potentially designers can change it say to orange or green :)

Better names: accent-color, moyo-blue-color (it refers to a bran book), etc